docs/FAQ.md

```diff
@@ -276,14 +276,15 @@ Theoretically yes, the future is not really projectable beyond 80 years. However
 
-We currently have a desktop [wallet](https://github.com/alephium/desktop-wallet/releases/latest) and the team is working on mobile and [web extension wallets](https://github.com/alephium/extension-wallet/releases/latest). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
+We currently have a [desktop wallet](https://github.com/alephium/desktop-wallet/releases/latest) and a [web extension wallet](https://github.com/alephium/extension-wallet/releases/latest). The team is actively working on a [mobile wallet](https://github.com/alephium/mobile-wallet). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
 
 ### When importing my seed to another desktop wallet, is there a way to import all generated addresses with it?
 
-At the moment, when using the desktop wallet, one will have to manually re-generate each address when restoring a mnemonic into a new wallet. 
-Assuming a wallet had generated X active addresses (an address is considered active when it has had at least one transaction), when the mnemonic of this wallet is restored and X addresses are generated, those addresses will be the same addresses as before and will have the same balance.
+Yes! When restoring a wallet, the app will automatically start a scanning process to find addresses that are "active" (active = have been used to send/receive transactions), so that the user doesn't have to manually re-generate them.
 
-Automatic address discovery will be added in the near future. That way, when restoring a wallet, the app will automatically start a scanning process to find addresses that are "active" (active = have been used to send/receive transactions), so that the user doesn't have to manually re-generate them.
+### What analytics does the desktop wallet collect?
+
+In Alephium, we take your concerns about privacy and user experience seriously. Enabling analytics can actually help us improve your experience without compromising your privacy. Rest assured that the information we collect is completely anonymous. When you launch your wallet for the first time, we generate a unique ID (for example, `vCJGCsDPrZ8WJaIKZMWjU`) which is the only identification information we require. We don't collect IPs or any other [personal data](https://posthog.com/blog/what-is-personal-data-pii). We only collect data on button clicks, number of wallets, number of addresses, number of contacts, and wallet preferences. This helps us understand which features are useful to our users and which ones are not. We encourage you to check out our open-source code base and [look for the `posthog?.capture` keyword](https://github.com/alephium/desktop-wallet/search?q=posthog%3F.capture) to see exactly which events we capture.
```

Comment:
Could we rephrase without using "we"? Like this for example: 

Alephium takes concerns about privacy and user experience seriously. Enabling analytics can actually help improve your experience without compromising your privacy. The information collected by the Desktop wallet is completely anonymous. When you launch your wallet for the first time, a unique ID is generated (for example, `vCJGCsDPrZ8WJaIKZMWjU`) which is the only identification information required. IPs or any other [personal data](https://posthog.com/blog/what-is-personal-data-pii) are not collected. The only data collected are button clicks, number of wallets, number of addresses, number of contacts, and wallet preferences. It helps understand which features are useful to users and which ones are not. Alephium encourages everyone to check out its open-source code base and [look for the `posthog?.capture` keyword](https://github.com/alephium/desktop-wallet/search?q=posthog%3F.capture) to see exactly which events are captured.